layout_padded/layout_padded.md

```diff
@@ -1516,21 +1618,47 @@ is representable as a value of type `size_t`,
 and is representable as a value of type `index_type`.
 
 ```c++
-static constexpr size_t @_static-padding-stride_@ = /* see-below */; // exposition only
+static constexpr size_t @_static-padding-stride_@ = /* @_see-below_@ */; // @_exposition only_@
 ```
 
 [4]{.pnum} The value is
 
-  * [4.1]{.pnum } `0`, if `extents_type::rank()`
+  * [4.1]{.pnum} `0`, if `extents_type::rank()`
       equals zero or one; otherwise
 
-  * [4.2]{.pnum } the `size_t` value which is
+  * [4.2]{.pnum} `dynamic_extent`, if `padding_value` or `extents_type::static_extent(0)` equals `dynamic_extent`; otherwise
+
+  * [4.3]{.pnum } the `size_t` value which is
     the least multiple of `padding_value`
     that is greater than or equal to
-    `extents_type::static_extent(0)`, if
-      `padding_value` does not equal `dynamic_extent` and `extents_type::static_extent(0)` does not equal `dynamic_extent`; otherwise
+    `extents_type::static_extent(0)`.
+
+<!--
+NOTE (mfh 2024/01/18) Previously, we specified _`stride-1`_
+as having type `extents<index_type, `_`static-padding-stride`_`>`.
+LWG preference from review on 2024/01/17
+is instead to specify it as having type `index_type`,
+and then to use a "Recommended practice" clause
+to highlight the performance optimization
+of using `extents` instead of `index_type`.
 
-  * [4.3]{.pnum } `dynamic_extent`.
+[[stacktrace.basic.nonmem]](https://eel.is/c++draft/stacktrace.basic.nonmem)
+and [[time.parse]](https://eel.is/c++draft/time.parse)
+give recent examples of "Recommended practice" clauses.
+
+[[description]](http://eel.is/c++draft/description) does not explain
+how to order a "Recommended practice" clause
+with respect to other library description clauses.
+-->
+
+```c++
+index_type /* @_see below_@ */ @_stride-1_@ = @_static-padding-stride_@; // @_exposition only_@
+```
+
+[5]{.pnum} *Recommended practice*: Implementations should not store
+this value if _`static-padding-stride`_ is not `dynamic_extent`.
+For example, using `extents<index_type,`_`static-padding-stride`_`>`
+instead of `index_type` as the type of _`stride-1`_ would achieve this.
```

Comment:
LWG mailing list suggestion (please change manually, not through github):

```suggestion
Using `extents<index_type,`_`static-padding-stride`_`>`
instead of `index_type` as the type of _`stride-1`_ would achieve this.
```

Reply:
Fixed.